website/static/js/tests/registrationModal.test.js

```diff
@@ -104,4 +107,90 @@ describe('registrationModal', () => {
+        it('returns true for date more than 2 days in the future', () => {
+            var validDateTwoDays = new Date(2016, 0, 1);
+            vm.pikaday(new Date (2016, 0, 4));
+            assert.isTrue(vm.minimumTimeValidation(null, null, validDateTwoDays));
+        });
+        it('returns true for date less than 4 years in the future', () => {
+            var validDateFourYears = new Date(2016, 0, 1);
+            vm.pikaday(new Date (2019, 11, 30));
+            assert.isTrue(vm.maximumTimeValidation(null, null, validDateFourYears));
+        });
+        it('returns false for date less than 2 days in the future', () => {
+            var invalidDateTwoDays = new Date(2016, 0, 1);
+            vm.pikaday(new Date (2016, 0, 2));
+            assert.isFalse(vm.minimumTimeValidation(null, null, invalidDateTwoDays));
+        });
+        it('returns false for date at least 4 years in the future', () => {
+            var invalidDateFourYears = new Date(2016, 0, 1);
+            vm.pikaday(new Date (2020, 0, 1));
+            assert.isFalse(vm.maximumTimeValidation(null, null, invalidDateFourYears));
+        });
+        it('returns true for date more than 2 days in the future, regardless of time zone', () => {
+            var validDateTwoDaysTZ = new Date(2016, 0, 1);
+            validDateTwoDaysTZ.setMinutes(validDateTwoDaysTZ.getMinutes() - validDateTwoDaysTZ.getTimezoneOffset());
+            var validDateTZFuture = new Date(validDateTwoDaysTZ);
+            validDateTZFuture.setDate(validDateTwoDaysTZ.getDate() + 3);
+            vm.pikaday(validDateTZFuture);
+            assert.isTrue(vm.minimumTimeValidation(null, null, validDateTwoDaysTZ));
+        });
+        it('returns true for date less than 4 years in the future, regardless of time zone', () => {
+            var validDateFourYearsTZ = new Date(2016, 0, 1);
+            validDateFourYearsTZ.setMinutes(validDateFourYearsTZ.getMinutes() - validDateFourYearsTZ.getTimezoneOffset());
+            var validDateTZFutureYears = new Date(validDateFourYearsTZ);
+            validDateTZFutureYears.setDate(validDateTZFutureYears.getDate() + 1459);
+            vm.pikaday(validDateTZFutureYears);
+            assert.isTrue(vm.maximumTimeValidation(null, null, validDateFourYearsTZ));
+        });
+        it('returns false for date less than 2 days in the future, regardless of time zone', () => {
+            var invalidDateTwoDaysTZ = new Date(2016, 0, 1);
+            invalidDateTwoDaysTZ.setMinutes(invalidDateTwoDaysTZ.getMinutes() - invalidDateTwoDaysTZ.getTimezoneOffset());
+            var invalidDateTZFuture = new Date(invalidDateTwoDaysTZ);
+            invalidDateTZFuture.setDate(invalidDateTwoDaysTZ.getDate() + 1);
+            vm.pikaday(invalidDateTZFuture);
+            assert.isFalse(vm.minimumTimeValidation(null, null, invalidDateTwoDaysTZ));
+        });
+        it('returns false for date at least 4 years in the future, regardless of time zone', () => {
```

Comment:
I think 'regardless of time zone' might not be the right thing to say here. Can you rephrase, or explain if the phrasing is exactly perfect?